resources/views/tiptap-editor.blade.php

```diff
@@ -45,6 +45,7 @@ class="relative z-0 tiptap-wrapper bg-white dark:bg-gray-700 rounded-md"
             x-on:insert-source.window="$event.detail.statePath === '{{ $statePath }}' ? insertSource($event.detail.source) : null"
             x-on:insert-grid-builder.window="$event.detail.statePath === '{{ $statePath }}' ? insertGridBuilder($event.detail.data) : null"
             x-on:update-editor-content.window="$event.detail.statePath === '{{ $statePath }}' ? updateEditorContent($event.detail.content) : null"
+            x-on:refresh-tiptap-editors.window="refreshEditorContent()"
```

Comment:
This needs the statePath check like the other window listeners, otherwise all editor instances would run the refresh on the event.

Reply:
This is intentional, I'm indeed looking for a way to broadcast a refresh event to all instances. A bit more context:

What's added in https://github.com/awcodes/filament-tiptap-editor/pull/129 works well on a resource with a top-level Tiptap field, I can refresh its UI because I know the statepath (e.g. `data.content`). On another resource, I'm using a top-level Builder field, with a few different block types, one of which is a Paragraph block using Tiptap. The issue I'm facing when I'm tryng to dynamically update the Builder data is that there is no easy way to know the nested statepaths of the Tiptap editors within. This is where the idea of broadcasting a "global" event came from.